enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/identity/ClusterBinder.java

```diff
@@ -54,11 +55,12 @@
 
     private ClusterId clusterId;
 
-    public ClusterBinder( SimpleStorage<ClusterId> clusterIdStorage, CoreTopologyService topologyService,
-                            LogProvider logProvider, Clock clock, ThrowingAction<InterruptedException> retryWaiter,
-                            long timeoutMillis, CoreBootstrapper coreBootstrapper, String dbName, int minCoreHosts )
+    public ClusterBinder( SimpleStorage<ClusterId> clusterIdStorage, SimpleStorage<DatabaseName> dbNameStorage, LogProvider logProvider, Clock clock,
+            ThrowingAction<InterruptedException> retryWaiter, long timeoutMillis, CoreBootstrapper coreBootstrapper, String dbName, int minCoreHosts,
+            CoreTopologyService topologyService )
```

Comment:
Why move `topologyService`? That and reformatting makes the diff a bit harder to read.